### ISSUE TITLE
Missing field in accessibility snapshot

### ISSUE THREAD

**Issue by mayur-waghela** [microsoft/playwright-mcp#501]
I am working with version 0.0.26 and came across with a very strange issue.
My test application is same and until yesterday the snapshot was working fine.

Today i started seeing the the Snapshot doesn't have few fields, although they are visible on UI.

For example the "Signatory List"

![Image](https://github.com/user-attachments/assets/ffbaf96c-4e02-4def-bb35-3afc8a61b0c1)

The snapshot shows as below. Only the text field is seen and Labels are gone. Until yesterday it was working fine and with respect to the test application nothing changed.

![Image](https://github.com/user-attachments/assets/bbc1e8aa-af2e-4de6-a180-263bf413400b)

**Comment by mayur-waghela** [comment]
does 0.0.27 has this bug ? even if it was updated to 0.0.27 , it should have shown a similar accessibility snapshot right ?

**Comment by pavelfeldman** [comment]
Not sure why you are asking me. Please file something we can reproduce, otherwise this thread is not converging.
